tests/integration/iop-externalistiod-remote-integration-test-defaults.yaml

```diff
@@ -0,0 +1,27 @@
+# This file provides some defaults for integration testing.
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  name: install
+spec:
+  meshConfig:
+    accessLogFile: "/dev/stdout"
+    defaultConfig:
+      proxyMetadata:
+        ISTIO_META_DNS_CAPTURE: "true"
```

Comment:
I do not like copying these in 4 places, they will fall out of sync (actually I think its starting out of sync?). istioctl can take multiple IOPs, we should do `-f base.yaml -f remote.yaml` so we get the defaults?